Application/EdFi.Ods.AdminApi/Infrastructure/OdsSecurityVersionResolver.cs

```diff
@@ -40,24 +30,21 @@ public OdsSecurityVersionResolver(IOdsApiValidator apiValidator, string apiServe
 
     private EdFiOdsSecurityModelCompatibility InitializeModelVersion()
     {
-        var validationResult = _apiValidator.Validate(_apiServerUrl).GetAwaiter().GetResult();
-
-        if (!validationResult.IsValidOdsApi || validationResult.Version == null)
+        try
         {
-            _log.Error("Unable to determine security model from ODS API Response");
-            throw validationResult.Exception ?? new Exception("No version reported from the Ed-Fi API");
+            return _odsApiVersion switch
+            {
+                "5.3" => EdFiOdsSecurityModelCompatibility.ThreeThroughFive,
+                "5.3-cqe" => EdFiOdsSecurityModelCompatibility.FiveThreeCqe,
+                "6.0" => EdFiOdsSecurityModelCompatibility.Six,
+                "6.1" => EdFiOdsSecurityModelCompatibility.Six,

```

Comment:
@simpat-adam  A better option when you need to return the same value for multiple options is using the "or" operator

`"6.0" or "6.1" => EdFiOdsSecurityModelCompatibility.Six,`

the default option has been replaced by "_"

```
{
                "5.3" => EdFiOdsSecurityModelCompatibility.ThreeThroughFive,
                "5.3-cqe" => EdFiOdsSecurityModelCompatibility.FiveThreeCqe,
                "6.0" or "6.1" => EdFiOdsSecurityModelCompatibility.Six,
                _ => throw new Exception()
}
```